resources/popup.js

```diff
@@ -1,27 +1,29 @@
+// Use the toggle button on the popup to switch the loggingEnabled option.
+function toggle() {
+    chrome.storage.sync.get({
+        loggingEnabled: true
+    }, function (res) {
+        chrome.storage.sync.set({
+            loggingEnabled: !(res.loggingEnabled)
+        })
+    });
+}
+
 $(document).ready(function () {
```

Comment:
This contains quite a large function, could these be split off into different functions?